src/_utils/checkPropertyExists.js

```diff
@@ -0,0 +1 @@
+export default (object, property) => Object.prototype.hasOwnProperty.call(object, property);
```

Comment:
Instead of creating this method, you can use [lodash' has method](https://lodash.com/docs/4.17.10#has) to check if object has a property